### PR TITLE
Use GUID, instead of plugin name, as key for plugin settings.

### PR DESCRIPTION
**Pull request by mast-eu**
This change was implemented in Git Extensions with PR https://github.com/gitextensions/gitextensions/pull/8995.

This PR is to align the Plugin Manager accordingly.

### CODE REVIEW

File: src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj.user [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <GitExtensionsDownloadPath>..\..\..\gitextensions.shared</GitExtensionsDownloadPath> <!-- path is relative to $(ProjectDir) -->
     <GitExtensionsReferenceVersion>latest</GitExtensionsReferenceVersion> <!-- 'latest' or 'v3.1' (= tag from GitHub releases) or 'v3.1.0.5877' (= build number from AppVeyor)-->
-    <GitExtensionsReferenceSource>GitHub</GitExtensionsReferenceSource> <!-- 'GitHub' or 'AppYevor' -->
+    <GitExtensionsReferenceSource>AppVeyor</GitExtensionsReferenceSource> <!-- 'GitHub' or 'AppVeyor' -->
```
Comment (mast-eu) [inline src/GitExtensions.PluginManager/GitExtensions.PluginManager.csproj.user:5]: Strictly speaking, this change would  not be part of this PR. It is to correct an old error, introduced by myself:
The master branch of a plugin should be pointing to the GE master branch.

**Review by ivangrek — APPROVED** [review]
👍

**Review by maraf — APPROVED** [review]

**Review by RussKie — APPROVED** [review]
